help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,66 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity Resolution source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+
+## Connect to the destination {#connect}
```

Comment:
the rest of the documentation looks like a copy-paste from the Amazon S3 destination documentation. if the destination exports files to an Amazon S3 bucket, that's fine, but it needs to be called out clearly. 

many other sections are missing, like how to verify if the data has made it through correctly to the destination